src/pages/OnboardingPage/OnboardingPage2.jsx

```diff
@@ -0,0 +1,62 @@
+import { useContext } from "react";
+import "./OnboardingPage.scss";
+import Button from "../../components/Button/Button";
+import { PageContext } from "../../contexts/PageContext";
+import loadingInactiveIcon from "../../assets/images/loadingInactiveIcon.svg";
+import loadingActiveIcon from "../../assets/images/loadingActiveIcon.svg";
+import arrowLeft from "../../assets/images/arrowLeft.svg";
+
+const OnboardingPage2 = () => {
+  const { handleNext, handleBack, progressBarIndex } = useContext(PageContext);
+  return (
+    <div className="onboarding-page">
+      <div className="onboarding-page__container">
+        <>
+          <div className="left-arrow-container" onClick={handleBack}>
+            <img src={arrowLeft} alt="arrowLeft" className="arrow-left" />
+          </div>
+          <div className="onboarding-page__text-container">
+            <div className="welcome-message">How good are you at field?</div>
+          </div>
+          <div className="onboarding-button-container">
+            <Button text="Beginner" color="white" eventListener={handleNext} />
+            <Button
+              text="Intermediate"
+              color="white"
+              eventListener={handleNext}
+            />
+            <Button text="Expert" color="white" eventListener={handleNext} />
+          </div>
+          <div className="loading-icon-container">
+            <img
```

Comment:
this is really weird. So if it's on that page there's a loading spinner? If there isn't then it's loading inactive? 

I think it's fine for now if you don't have the time but this is really specific logic that you would have to finnick around with if you had more than 4 pages (constantly add or delete image)

You can build a loop with an argument max pages. It will increment by 1 and if the number === progressBarIndex then set the loadingActiveIcon true.